lib/installer.js

```diff
@@ -36,12 +36,12 @@ async function downloadWAD () {
 }
 
 async function installWAD (msiPath) {
-  log.info("Running MSI installer...");
+  log.info(`Running MSI installer...`);
   await exec('msiexec', ['/i', msiPath, '/qn']);
 }
 
 async function verifyWAD () {
-  log.info("Verifying WinAppDriver is installed with correct checksum");
+  log.info(`Verifying WinAppDriver version ${WAD_VER} is installed via comparing the checksum.  You must use WinAppDriver version ${WAD_VER}`);
```

Comment:
can we split the long string into two lines?

Reply:
done